src/resources/js/views/month-mobile-events.js

```diff
@@ -69,12 +74,55 @@ tribe.events.views.monthMobileEvents = {};
 			if ( contentId ) {
 				$content = $container.find( '#' + contentId );
 				tribe.events.views.accordion.closeAccordion( $header, $content );
+				hasDays = true;
 			}
 
 			obj.closeMobileEvents( $header, $content );
 		} );
 	};
 
+	/**
+	 * Handles the user selection of a day in the Grid.
+	 *
+	 * @since TBD
+	 *
+	 * @param {bool} viewHasDays Whether the day selected by the user has days in it or not.
+	 *
+	 * @return {void} The method does not return a value and will hide and show message elements.
+	 */
+	obj.handleMobileDayClick = function( hasDays ) {
+		var $defaultNotices = $( obj.selectors.mobileEventsDefaultNotices );
+		var $dynamicNotices = $( obj.selectors.mobileEventsDynamicNotices );
+
+		if ( hasDays ) {
+			$defaultNotices.hide();
+			$dynamicNotices.hide();
```

Comment:
Instead of using `hide()` and `show()` methods, let's add and remove the class `tribe-common-a11y-hidden`. We can add it to the selectors above and do something like `$defaultNotices.addClass( obj.selectors.hidden.className() )`